engine/apps/schedules/models/shift_swap_request.py

```diff
@@ -181,7 +181,7 @@ def organization(self) -> "Organization":
 
     @property
     def possible_benefactors(self) -> QuerySet["User"]:
-        return self.schedule.related_users().exclude(pk=self.beneficiary_id)
+        return self.schedule.users_in_future_schedule().exclude(pk=self.beneficiary_id)
```

Comment:
I am taking the users from events in the future considering 30days should we rethink on this thought ?